client/e2e/poms/profile.ts

```diff
@@ -46,26 +46,20 @@ export class ProfilePOM {
     await fieldsUpdate(this.page);
     // Click the Submit button
     await this.buttonSubmit.click();
-    // Wait for the success message to appear
-    await this.page.waitForSelector('div:has-text("✅ Success")', {
-      timeout: 15000,
+    // Response from submit either shows errors or triggeres handleSubmit which handles state changes on submit button etc.
+    // 🔍 Assert that the error selector is not available
+    await this.page.waitForSelector(DataTestID.ERROR_PROFILE, {
```

Comment:
`waitForSelector` is discouraged: https://playwright.dev/docs/api/class-page. Playwright's supposed to have good async/waiting support--would something as simple as page.getByText("✅ Success") work? Or is this one more complicated because it's state-related?

Reply:
@BCerki  I believe the issue is with a race condition for new user having a successful submit and getting re-directed to dashboard from profile.

Reply:
Sorry, I think I put the wrong link, if we have to use a selector that involves waiting, the recommendation is `locator.waitFor()` instead of `waitForSelector`
![image](https://github.com/bcgov/cas-registration/assets/77306764/d246b994-bb94-475c-aab3-b0f0d7779f64)

Reply:
@BCerki  for your review:
https://github.com/bcgov/cas-registration/pull/1103/commits/8ccde178e4d05186b94f5c06f4092dda898f50a0